src/main/java/com/example/demo/repository/Tournament.java

```diff
@@ -0,0 +1,31 @@
+package com.example.demo.repository;
+
+import com.example.demo.entity.Team;
+import com.example.demo.service.SQLMethods;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@AllArgsConstructor
+public class Tournament {
+    private TeamsList teamsList;
+    SQLMethods sqlMethods;
+
+    public String getWinner(Team teamFirst, Team teamSecond, String round) {
+
+        String result = round + ", " + teamFirst.getTeamName() + ",  " + teamSecond.getTeamName() + ", " + teamFirst.getPoints() + " : " + teamSecond.getPoints();
```

Comment:
If you need to construct long String I recommend to use either `java.lang.StringBuilder` either `java.lang.String#format(java.util.Locale, java.lang.String, java.lang.Object...)`

Reply:
With `String#format` it will be easier to predict output.